### ISSUE TITLE
python manage.py dbbackup gives error 

### ISSUE THREAD

**Issue by vaibhav-jain** [django-dbbackup/django-dbbackup#141]
In the docs it says use this command `python manage.py dbbackup`  to test that everything is working fine, But after installation when I tried running this command I got this error 
```
(mailengine)prime@ubuntu:~/workspace/mailengine$ python manage.py dbbackup maildb
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/dbbackup/utils.py", line 107, in wrapper
    body = reporter.get_traceback_html()
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/views/debug.py", line 384, in get_traceback_html
    return t.render(c)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/base.py", line 209, in render
    return self._render(context)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/base.py", line 201, in _render
    return self.nodelist.render(context)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/base.py", line 903, in render
    bit = self.render_node(node, context)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/debug.py", line 79, in render_node
    return node.render(context)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/debug.py", line 89, in render
    output = self.filter_expression.resolve(context)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/base.py", line 674, in resolve
    new_obj = func(obj, *arg_vals)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/template/defaultfilters.py", line 771, in date
    return formats.date_format(value, arg)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/formats.py", line 136, in date_format
    return dateformat.format(value, get_format(format or 'DATE_FORMAT', use_l10n=use_l10n))
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/formats.py", line 110, in get_format
    for module in get_format_modules(lang):
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/formats.py", line 82, in get_format_modules
    modules = _format_modules_cache.setdefault(lang, list(iter_format_modules(lang, settings.FORMAT_MODULE_PATH)))
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/formats.py", line 51, in iter_format_modules
    if not check_for_language(lang):
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/translation/__init__.py", line 181, in check_for_language
    return _trans.check_for_language(lang_code)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/lru_cache.py", line 125, in wrapper
    result = user_function(*args, **kwds)
  File "/home/prime/.virtualenvs/mailengine/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 409, in check_for_language
    if not language_code_re.search(lang_code):
TypeError: expected string or buffer
```
But after setting up the location settings it worked. So either documentation needs update or code is broken somewhere.

```
DBBACKUP_STORAGE = 'dbbackup.storage.filesystem_storage'
DBBACKUP_STORAGE_OPTIONS = {'location': '/my/backup/dir/'}
```

**Comment by ZuluPro** [comment]
Thank you very much for this, I'll check it.

Anthony Monthe - Open source enthousiaste
anthony-monthe.me

**Comment by vaibhav-jain** [comment]
Thank you for this useful tool.  Glad to help.

**Comment by ZuluPro** [comment]
Are you sure this error come from dbbackup ?
Can you try to launch another command, and launch it without dbbackup ?

**Comment by vaibhav-jain** [comment]
Hi,

Actually I think the error occurred because settings were missing . . . .  after setting the 
```
DBBACKUP_STORAGE = 'dbbackup.storage.filesystem_storage'
DBBACKUP_STORAGE_OPTIONS = {'location': '/my/backup/dir/'}
```
It worked. 
But in docs it is not mentioned clearly that these settings are compulsory. Please update the docs. Rest if fine

Thanks,
Vaibhav

**Comment by gipi** [comment]
+1: pip installed version 2.4.1 and without the configuration indicated from @vaibhav-jain raise an exception.

BTW, the releases are not tagged on Github, so I cannot do a fix by myself.

**Comment by ZuluPro** [comment]
This issue can be close with this merged PR #155
